main.html

```diff
@@ -134,122 +177,204 @@ <h1 class="let-ride__header">Let's ride</h1>
+                        <section class="ride__rentals">
+                            <span class="ride__card-header">Rentals</span>
+                            <img src="Lyft__car.png" class="ride__car" alt="Ride__Rentals"/>
+                            <div class="ride__card-points">
+                                <div class="ride__card-point">
+                                    <img src="car-key.png" class="ride__keys" alt="ride__keys"/>
+                                    <p>Seamless</p>
+                                </div>
+                                <div class="ride__card-point">
+                                    <img src="earth.png" class="ride__earth" alt="ride__earth"/>
+                                    <p>Independence</p>
+                                </div>
+                            </div>
+                        </section>
+                    </div>
+                    <div class="ride__card ride__card--bikes">
+                        <section class="ride__rentals">
+                            <span class="ride__card-header">Bikes & Scooters</span>
+                            <img src="Lyft__car.png" class="ride__bike" alt="Ride__Rentals"/>
+                            <div class="ride__card-points">
+                                <div class="ride__card-point">
+                                    <img src="thunder.png" class="ride__keys" alt="ride__keys"/>
+                                    <p>Efficient</p>
+                                </div>
+                                <div class="ride__card-point">
+                                    <img src="ec-friendly.png" class="ride__earth" alt="ride__earth"/>
+                                    <p>Eco-friendly</p>
+                                </div>
+                            </div>
+                        </section>
+                    </div>
+                    <div class="ride__card ride__card--wait">
+                        <section class="ride__rentals">
+                            <span class="ride__card-header">Wait & Save </span>
+                            <img src="Lyft__car.png" class="ride__car" alt="Ride__Rentals"/>
+                            <div class="ride__card-points">
+                                <div class="ride__card-point">
+                                    <img src="money.png" class="ride__keys" alt="ride__keys"/>
+                                    <p>Budget-friendly</p>
+                                </div>
+                                <div class="ride__card-point">
+                                    <img src="man.png" class="ride__earth" alt="ride__earth"/>
+                                    <p>Private</p>
+                                </div>
+                            </div>
+                        </section>
+                    </div>
+                    <div class="ride__card ride__card--lyft">
+                        <section class="ride__rentals">
```

Comment:
I'd not use the section here, [read a bit more about it](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/section)

Reply:
+